map/map_loader/src/pointcloud_map_loader/differential_map_loader_module.cpp

```diff
@@ -63,10 +64,27 @@ bool DifferentialMapLoaderModule::onServiceGetDifferentialPointCloudMap(
   GetDifferentialPointCloudMap::Request::SharedPtr req,
   GetDifferentialPointCloudMap::Response::SharedPtr res)
 {
+  const int64_t map_upper_limit_margin = 10000;
```

Comment:
what is the unit? (size?)

Reply:
I removed map_upper_limit_margin and added description in https://github.com/autowarefoundation/autoware.universe/commit/ae4e410edb2f66f34d20cd4d07a47e8d5d8b9124